operations/security_and_authentication/Change_Credentials_on_ServerTech_PDUs.md

```diff
@@ -17,143 +34,177 @@ This procedure changes password used by the `admn` user on ServerTech PDUs. Eith
     ```
 
     Sample output:
-    ```
+
+    ```text
     x3000m0
     ```
 
-2.  Specify the existing password for the `admn` user:
+1. Set up aliases:
+
+    ```bash
+    ncn-m001# VAULT_PASSWD=$(kubectl -n vault get secrets cray-vault-unseal-keys -o json | jq -r '.data["vault-root"]' |  base64 -d)
+    ncn-m001# alias vault='kubectl -n vault exec -i cray-vault-0 -c vault -- env VAULT_TOKEN=$VAULT_PASSWD VAULT_ADDR=http://127.0.0.1:8200 VAULT_FORMAT=json vault'
+    ```
+
+1. Specify the existing password for the `admn` user:
+
+    To extract the global credentials from vault for the PDUs:
+
+    ```bash
+    vault kv get secret/pdu-creds/global/pdu
```

Comment:
Add `ncn-m001# ` to the beginning of the command

Reply:
updated